charts/spiderpool/templates/pod.yaml

```diff
@@ -1,159 +1,116 @@
 {{- if or .Values.ipam.enableIPv4 .Values.ipam.enableIPv6 -}}
-{{- if or .Values.clusterDefaultPool.installIPv4IPPool .Values.clusterDefaultPool.installIPv6IPPool .Values.coordinator.enabled -}}
-apiVersion: v1
-kind: Pod
+{{- if or .Values.clusterDefaultPool.installIPv4IPPool .Values.clusterDefaultPool.installIPv6IPPool .Values.coordinator.enabled }}
```

Comment:
these premises seems not to be right ,  because the job also help install multus config

Reply:
done.